jonprl-mode.el

```diff
@@ -48,6 +48,28 @@
   :type 'file
   :group 'jonprl)
 
+(defcustom jonprl-pretty-symbols
+  '(("*" . ?×)
+   ("->" . ?→)
+   (":" . ?∶)
+   ("<" . ?⟨)
+   ("<>" . ?⋄)
+   ("=>" . ?⇒)
+   ("=def=" . ?≝)
+   (">" . ?⟩)
+   ("[" . ?⸤)
+   ("]" . ?⸥)
```

Comment:
Yeah, it may not be worth doing anything with the delimiters. It does replace the others (not followed by `.`) but I suppose it could be confusing.